Geometry/Records/interface/TrackerDigiGeometryRecord.h

```diff
@@ -36,7 +20,8 @@ class TrackerDigiGeometryRecord :
                 TrackerAlignmentErrorRcd,
```

Comment:
@ianna 
Can you remove the dependency from `TrackerAlignmentErrorRcd`? It is now removed from GT.